lib/zendesk_apps_tools/command.rb

```diff
@@ -191,13 +191,67 @@ def update
       deploy_app(:put, "/api/v2/apps/#{app_id}.json", {})
     end
 
+    desc 'migrate', 'Helps with the migration of a v1 app to v2'
+    shared_options(except: [:unattended, :clean])
+    method_option :auto, default: false, required: false, type: :boolean, aliases: '-a'
+    def migrate
+      cache.clear
+      setup_path(options[:path])
+      @command = 'Migrate'
+      unless migration_helper_installed
+        try_install = get_value_from_stdin("The Zendesk App Migration Helper isn't installed yet. Would you like to try installing now?", limited_to: ['y', 'yes', 'n', 'no'], default: 'y' )
+        say_error_and_exit("Please install the Zendesk App Migration Helper before running this command") if (try_install =~ /^y(es)?$/).nil? # thats a no
+        say_error_and_exit("Unable to install the Zendesk App Migration Helper \
+                            Please follow the installation instructions at \
+                            https://github.com/zendesk/zendesk_app_migrator \
+                            before running this command again") unless install_migration_helper
+      end
+      migrate_app(options)
+    end
+
     desc "version, -v", "Print the version"
     def version
       say ZendeskAppsTools::VERSION
     end
 
     protected
 
+    def migration_helper_installed
+      is_semver(app_migrator_version)
+    end
+
+    def node_js_installed
+      !!system("npm -v", out: File::NULL)
```

Comment:
do we have any requirements for node/npm version?

Reply:
Good question.  I'd incorrectly assumed that the `.node-version` config would be respected when the package was globally installed.  This not being the case, I feel like using the [`engines` field in package.json](https://docs.npmjs.com/files/package.json#engines) might be the best approach.  Would you agree?